### DISCUSSION TITLE
How to cut a square picture into a circle

### DISCUSSION THREAD

**Discussion by qiaoabcd** [photonstorm/phaser#6071] [Q&A]
The images THAT I use in the game come from the web, and they're all square,  How do I crop an image to a circle once it's loaded into the game, or is there a way to crop it to a circle before loading it

**Comment by spayton** [comment]
To do this in Phaser, you need to looking at 'masking' in some way.  Either apply a circular mask to your squares when drawing (every screen update), or pre-render the masked image to another texture and use that.

This might be useful: [https://phaser.discourse.group/t/pre-render-images-with-masks-to-texture/5852](url)
